docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -0,0 +1,98 @@
+
+## Enable Einstein for Developers
+
+An Admin can enable Einstein for Developers in a supported Salesforce org by following the given steps.
+
+1. From Setup, enter Einstein for Developers in the Quick Find box, then select Einstein for Developers. If you don’t see Einstein for Developers as an option under Setup in your Developer Edition org, it was likely created before Einstein for Developers was introduced and doesn’t have the correct permissions. To continue, create another Developer Edition org.
+2. Enable Einstein for Developers, and then review and accept the license agreement.
+
+Einstein for Developers(beta) is activated in the org and all the users in the org have access to the feature.
+
+### Add Users to the Org
+
+An admin can add additional users to the org as needed:
+
+1. From Setup, enter Users in the Quick Find box, then select **Users**.
+2. Click **New User** or **Add Multiple Users**.
+3. Select the appropriate license type and profile based on the user’s role.
+4. Select the **Generate passwords and notify user via email** checkbox.
+5. Click **Save**.
+   This procedure generates an email inviting the new users into the org.
+
+### Visual Studio Code Version
+
+VS Code releases a new version each month with new features and important bug fixes. You must be on VS Code Version 1.76 or higher to run the Einstein for Developers extension. You can manually check for updates from **Help** **> Check for Updates** on Linux and Windows or **Code > Check for Updates** on macOS.
+
+## Get Started with Einstein for Developers
+
+A developer with access to an org enabled for Einstein for Developers can install the Einstein for Developers extensions in VS Code, connect to org and start using the extension.
+
+## Install Einstein for Developers Extensions
+
+To use Einstein for Developerslocally, first install Salesforce Extensions and Salesforce CLI on your desktop.
+
+<a class="slds-button slds-button_brand landing__header-cta slds-m-vertical--x-large" href="https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode">Install the Salesforce Extensions</a>
+
+Then install Einstein for Developers(beta)from the VS Code marketplace.
+
+<a class="slds-button slds-button_brand landing__header-cta slds-m-vertical--x-large" href="https://marketplace.visualstudio.com/items?itemName=salesforcedx-einstein-gpt">Install Einstein for Developers(beta)</a>
+
+## Connect to an Org
+
+The Einstein for Developers(beta) tool is run in the context of a Salesforce org, in Salesforce DX project. To use this tool:
+
+1. Go to **File** > **Open Folder** in the menu and open an existing Salesforce DX project in VS Code, or create a new one.
+
+2. Run the **SFDX: Authorize an Org** command to connect to the Salesforce sandbox org or scratch org that has Einstein for Developers(beta) enabled.
+
+A sidebar panel opens and the Einstein logo on the activity bar confirms that the extension is installed.
```

Comment:
The sidebar won't open automatically - we should just say that they'll be able to tell the Einstein for Devs extension is installed and active via the Status Bar icon - and they'll see the Einstein icon in their sidebar as well